src/common/api/curriculum/quiz/requests.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+export interface Question {
+  question: string;
+  options: string[];
+  answer: number;
+}
+
+interface QuizMeta {
+  data: Question[];
+}
+
+export const getQuizMeta = async (quizId: number): Promise<QuizMeta> => {
```

Comment:
```suggestion
export const getQuizMeta = async (quizId: number): Promise<Question[]> => {
```

You're actually returning an `Promise<Question[]>` from `getQuizMeta` instead of an object with a `data` property that's a `Question[]`.
![image](https://user-images.githubusercontent.com/22752497/100694208-371f4e00-3354-11eb-9456-8f686ed5f5c8.png)